### PR TITLE
delegation: make the examples more examply, less testy

### PR DESCRIPTION
**Pull request by MichaelMure**

### CODE REVIEW

File: token/delegation/options.go [hunk 1]
```diff
@@ -24,6 +24,15 @@ func WithExpiration(exp time.Time) Option {
 	}
 }
 
+// WithExpirationIn set's the Token's optional "expiration" field to Now() plus the given duration.
+func WithExpirationIn(exp time.Duration) Option {
```
Comment (smoyer64) [inline token/delegation/options.go:28]: How about `WithExpirationAfter`?

File: token/delegation/examples_test.go [hunk 2]
```diff
@@ -123,40 +146,56 @@ func ExampleNew() {
...
-	// when the token is created.
-	nonce := []byte{0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0a, 0x0b}
-
-	tkn, err := delegation.Root(
-		issuerPrivKey,
-		audienceDID,
-		command,
-		policy,
-		delegation.WithNonce(nonce),
+	issPriv, issPub, err := crypto.GenerateEd25519Key(rand.Reader)
```
Comment (smoyer64) [inline token/delegation/examples_test.go:149]: How do the examples pass when you don't have fixed input?

File: token/delegation/options.go (hunk 1 above)
Comment (MichaelMure) [reply token/delegation/options.go:28]: reasonable, I'll change

File: token/delegation/examples_test.go (hunk 2 above)
Comment (MichaelMure) [reply token/delegation/examples_test.go:149]: it doesn't check the values, but that's ok. What we need is a "quick start", not a test.

**Pull request merged**